lib/TokenManager.ts

```diff
@@ -393,8 +494,27 @@ export class TokenManager {
         this.remove(key);
       }
     };
-    this.on('expired', onTokenExpiredHandler);
+    this.on(EVENT_EXPIRED, onTokenExpiredHandler);
 
     setExpireEventTimeoutAll(sdk, tokenMgmtRef, storage);
+
+    // Sync authState cross multiple tabs when localStorage is used as the storageProvider
+    window.addEventListener('storage', ({ key, newValue, oldValue }: StorageEvent) => {
+      const emitEventsAndResetExpireEventTimeouts = () => {
+        this._resetExpireEventTimeoutAll();
+        this._emitEventsForCrossTabsStorageUpdate(newValue, oldValue);
+      };
+
+      if (key !== options.storageKey) {
+        return;
+      }
+      // LocalStorage cross tabs update is not synced in IE, set a 1s timer to read latest value
+      // https://stackoverflow.com/questions/24077117/localstorage-in-win8-1-ie11-does-not-synchronize
+      if (isIE11OrLess() && newValue !== oldValue) {
+        setTimeout(() => emitEventsAndResetExpireEventTimeouts(), 1000);
+      } else {
+        emitEventsAndResetExpireEventTimeouts();
```

Comment:
should we check for `newValue !== oldValue` here?  I couldn't see anything saying that replacing a value with an equal value wouldn't fire the events

Reply:
Tested in chrome, looks like if the same value is added with `setItem`, then no event will be emitted.
But, I agree, it would be no hurt to use `newValue !== oldValue` as a general filter.